mrjob/options.py

```diff
@@ -526,9 +526,9 @@ def add_emr_opts(opt_group):
 
         opt_group.add_option(
             '--s3-endpoint', dest='s3_endpoint', default=None,
-            help=('Host to connect to when communicating with S3 (e.g.'
-                  ' s3-us-west-1.amazonaws.com). Default is to infer this from'
-                  ' region (see --aws-region).')),
+            help=('Force mrjob to connect to S3 on this endpoint (e.g.'
```

Comment:
Don't want to drop this option because I want to give people with a weird network or in a weird corner of AWS (e.g. GovCloud) a possible workaround.